code/controllers/subsystem/ambience.dm

```diff
@@ -11,7 +11,7 @@ SUBSYSTEM_DEF(ambience)
 /datum/controller/subsystem/ambience/fire(resumed)
 	for(var/client/client_iterator as anything in ambience_listening_clients)
 
-		if(isnull(client_iterator))
+		if(isnull(client_iterator) || isnewplayer(client_iterator.mob))
```

Comment:
Although I know little about this system, wouldn't the better fix to determine why new players are put in the ambience_listening_clients list in the first place?